docs/src/systems/weakly_compressible_sph.md

```diff
@@ -150,6 +150,44 @@ Pages = [joinpath("general", "corrections.jl")]
 
 
 ## [Surface Tension](@id surface_tension)
+
+### Akinci based intra-particle force surface tension and wall adhesion model
+The work by Akinci proposes three forces:
+- a cohesion force
+- a surface normal force
```

Comment:
```suggestion
### Akinci-based intra-particle force surface tension and wall adhesion model
The work by Akinci proposes three forces:
- a cohesion force
- a surface normal force
```
And shouldn't "surface normal force" be "curvature minimization force"? I find it confusing that you list three forces here and explain a different force below.